recirq/fermi_hubbard/publication.py

```diff
@@ -241,10 +241,10 @@ def fetch_publication_data(
 
     base_url = "https://datadryad.org/stash/downloads/file_stream/"
     data = {
-        "gaussians_1u1d_nofloquet": "706210",
-        "gaussians_1u1d": "706211",
-        "trapping_2u2d": "706212",
-        "trapping_3u3d": "706213"
+        "gaussians_1u1d_nofloquet": "451326",
+        "gaussians_1u1d": "451327",
+        "trapping_2u2d": "451328",
+        "trapping_3u3d": "451329"
```

Comment:
It's a bummer these aren't permanent identifiers. I reached out to dryad help and they essentially said the only thing permanent is the dataset DOI. So a long-term fix will probably need to locate files from the DOI instead of from /stash/downloads/file_stream.